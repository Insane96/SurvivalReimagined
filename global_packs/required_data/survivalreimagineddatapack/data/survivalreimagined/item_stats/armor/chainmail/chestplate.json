{
	"item": "minecraft:chainmail_chestplate",
	"durability": 48,
	"armor": 5,
	"modifiers": [

	]
}