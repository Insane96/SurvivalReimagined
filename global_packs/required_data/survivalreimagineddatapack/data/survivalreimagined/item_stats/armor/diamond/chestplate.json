{
	"item": "minecraft:diamond_chestplate",
	"durability": 240,
	"armor": 6,
	"modifiers": [

	]
}