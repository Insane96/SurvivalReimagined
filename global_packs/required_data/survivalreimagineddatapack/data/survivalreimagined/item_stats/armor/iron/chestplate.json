{
	"item": "minecraft:iron_chestplate",
	"durability": 112,
	"armor": 4,
	"modifiers": [

	]
}