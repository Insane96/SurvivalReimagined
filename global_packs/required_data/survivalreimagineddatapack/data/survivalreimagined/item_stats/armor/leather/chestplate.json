{
	"item": "minecraft:leather_chestplate",
	"durability": 80,
	"armor": 2,
	"modifiers": [

	]
}