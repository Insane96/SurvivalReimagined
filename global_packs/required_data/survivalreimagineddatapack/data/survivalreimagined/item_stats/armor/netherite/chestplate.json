{
	"item": "minecraft:netherite_chestplate",
	"durability": 240,
	"armor": 7,
	"modifiers": [

	]
}