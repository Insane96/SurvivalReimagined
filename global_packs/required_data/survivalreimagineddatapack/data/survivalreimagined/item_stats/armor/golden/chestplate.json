{
	"item": "minecraft:golden_chestplate",
	"durability": 64,
	"armor": 2,
	"modifiers": [

	]
}